src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -57,15 +57,16 @@ private sealed class ConsoleLogsSubscription
     [Parameter]
     public string? ResourceName { get; set; }
 
+    private readonly TaskCompletionSource _logViewerReadyTcs = new();
```

Comment:
RunContinuationsAsynchronously

Reply:
This is a case where that isn't desirable. The code after the log viewer is ready should run in the same context so it is on the UI thread.

Reply:
There's no UI thread. As long as the code is awaited the sync context will be preserved from the other side. The real question is whether the calling thread (the one calling Set result) wants to block inline while running the continuation.

Reply:
Blocking doesn't matter either way. I changed it.